frontend/src/streamlist/__tests__/StreamsTable.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import {
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  jest,
+  it
+} from "@jest/globals";
+import { Box, TableContainer } from "@material-ui/core";
+import MaterialTable from "material-table";
+import StreamsTable from "../StreamsTable";
+
+Enzyme.configure({ adapter: new Adapter() });
+jest.spyOn(global.console, "error");
+
+describe("<StreamsTable/> component", () => {
+  let wrapper;
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe("has the correct components", () => {
+    const dummyStreams = [];
+    beforeEach(() => {
+      wrapper = Enzyme.shallow(<StreamsTable streamDetails={dummyStreams} />);
+    });
+    it("contains one (1) Box component", () => {
+      expect(wrapper.find(Box)).toHaveLength(1);
+    });
```

Comment:
i get that you're probably being cheeky / humorous, but let's remove the (1)

Reply:
It was left like that in the DevicesTable test so it got copy-pasted into this version as well. Will probably fix both!